snapcraft/internal/repo/_platform.py

```diff
@@ -26,6 +26,7 @@
     '"elementary"',
     'debian',
     'neon',
+    'LinuxMint',
```

Comment:
seems it only has that info in 18, seemingly 17 has the ubuntu version of the file
https://bugs.launchpad.net/linuxmint/+bug/1641491